webpack.common.js

```diff
@@ -92,7 +92,8 @@ function generateHTML(){
   pages.unshift('index.html');
   const htmlPages=pages.filter(page => path.extname(page)=='.html');
   return htmlPages.map(item => {
-    const name=item.split((__dirname+'/src/pages/')).at(-1).split('.')[0];
+    const name=item.split((__dirname+'/src/pages/')).at(-1).split('.')[0].toString();
+    console.log(typeof(name))
```

Comment:
These changes were made when you were debugging local node issues.
Remove them since you resolved the issue locally and they are no longer needed.